docs/ai/skills/review-animations/STANDARDS.md

```
@@ -131,21 +105,17 @@ Legacy fallback: `useEffect(() => setMounted(true), [])` + `data-mounted` attrib
-.button:active .overlay {
-  transition: clip-path 2s linear;
-} /* press: slow, deliberate */
+.overlay { transition: clip-path 200ms ease-out; }            /* release: fast */
+.button:active .overlay { transition: clip-path 2s linear; }  /* press: slow, deliberate */
 ```
 
 ## Performance
 
 - **Only animate `transform` and `opacity`** — they skip layout/paint and run on the GPU. `padding`/`margin`/`height`/`width`/`top`/`left` trigger all three rendering steps.
 - **Don't drive child transforms via a CSS variable on the parent** — it recalcs styles for all children. Set `transform` directly on the element.
   ```js
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)